.github/workflows/CloudTesting.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       - name: Setup vcpkg
         uses: lukka/run-vcpkg@v11.1
         with:
-          vcpkgGitCommitId: 5e5d0e1cd7785623065e77eff011afdeec1a3574
+          vcpkgGitCommitId: b1b19307e2d2ec1eefbdb7ea069de7d4bcd31f01
```

Comment:
should this be ce613c41372b23b1f51333815feb3edd87ef8a8b? otherwise we are running this job against a different azure sdk potentially